frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -55,129 +59,123 @@ function AuthedView() {
-              </MenubarContent>
-            </MenubarMenu>
-          </Menubar>
-        </div>
-      </div>
+            <div className="md:w-fit w-full">
+              <Menubar className="border border-border md:w-fit w-full ">
+                <MenubarMenu>
+                  <MenubarTrigger className="flex md:w-fit w-full ring-secondary md:justify-center justify-between text-sm items-center gap-1">
+                    Status
+                    <ChevronsUpDown className="w-4" />
+                  </MenubarTrigger>
+                  <MenubarContent className=" border  w-[calc(var(--radix-menubar-trigger-width)+0.5rem)]  border-border md:min-w-6  md:w-auto">
+                    <MenubarContentItem icon={Rocket} text="Active" />
+                    <MenubarContentItem icon={Trash} text="Archived" />
+                  </MenubarContent>
+                </MenubarMenu>
+              </Menubar>
+            </div>
+          </div>
 
-      <Table>
-        <TableHeader className="bg-toggle">
-          <TableRow className="border-none">
-            <TableHead>Name</TableHead>
-            <TableHead className="flex items-center gap-2">
-              Last Updated
-              <ArrowDown size={15} />
-            </TableHead>
-            <TableHead>Status</TableHead>
-            <TableHead className="text-right">Actions</TableHead>
-          </TableRow>
-        </TableHeader>
-        <TableBody>
-          {projects.map((project) => (
-            <TableRow className="border-border cursor-pointer" key={project.id}>
-              <TableCell className="font-medium flex items-center gap-3">
-                <Folder size={18} />
-                {project.name}
-              </TableCell>
-              <TableCell>{project.updated_at}</TableCell>
-              <TableCell>
-                <StatusBadge color={getBadgeColor(project.tracker)}>
-                  {project.status}
-                </StatusBadge>
-              </TableCell>
-              <TableCell className="flex justify-end">
-                <div className="w-fit flex items-center gap-3">
-                  Settings
-                  <Settings width={18} />
-                </div>
-              </TableCell>
-            </TableRow>
-          ))}
-        </TableBody>
-      </Table>
+          <Table>
+            <TableHeader className="bg-toggle">
+              <TableRow className="border-none">
+                <TableHead>Name</TableHead>
+                <TableHead>Description</TableHead>
+                <TableHead className="flex items-center gap-2">
+                  Last Updated
+                  <ArrowDown size={15} />
+                </TableHead>
+                <TableHead>Status</TableHead>
+                <TableHead className="text-right">Actions</TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {projectList?.map((project: any) => (
```

Comment:
```suggestion
              {projectList.map((project) => (
```

We don't allow unnecessary `any` in this project, almost everything should be fully typed.